lib/evss/aws_creds.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'net/http'
+
+module EVSS
+  class AwsCreds
+    METADATA_ENDPOINT = 'http://169.254.169.254/latest/meta-data/iam/security-credentials/CloudWatchRole/'
```

Comment:
It's a link local address, will be the same for all environments that are on AWS instances
http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
Was recommended by devops to use this 
https://github.com/department-of-veterans-affairs/devops/issues/659